frontend/src/lib/components/PropertyFilters/PropertyFilter.js

```diff
@@ -205,7 +205,7 @@ export function PropertyFilter({ index, onComplete, logic }) {
             animated={false}
             style={{ minWidth: displayOperatorAndValue ? 700 : 350 }}
         >
-            <TabPane tab="Property" key="property" style={{ display: 'flex' }}>
```

Comment:
We can't remove flex here, because that'd break the layout of the Property row. Also, I don't _think_ this poses a problem, as everywhere but prod AntD cooperates with this prop just fine.